src/@dvcorg/gatsby-theme-iterative/components/LayoutHeader/Alert/content.tsx

```diff
@@ -0,0 +1,17 @@
+import Link from '@dvcorg/gatsby-theme-iterative/src/components/Link'
+import React from 'react'
+
+export const AlertContent = () => (
+  <>
```

Comment:
Is it expected that the deployed branch site doesn't contain the right alert content? 

<img width="1353" alt="image" src="https://github.com/iterative/dvc.org/assets/43496356/542cbc91-e216-475b-ba55-cb1a51378ec2">

Reply:
No, there should be a new banner. It must be because of `cache`.

Reply:
I have merged PRs related to `yarn.lock` file change and expect to rebuild without cache.

Reply:
Yes, that did the trick. Looks great!